docs/ReleaseNotes.md

```diff
@@ -1,6 +1,13 @@
 # Release Notes
 
-## Unreleased changes:
+# Important: .NET 4.0 support is currently **disabled**; if you need .NET 4.0, please stick with 1.2.1 while we keep investigating
```

Comment:
Yes. It's baseline against `StackExchange:NRediSearch` . I will close this PR and reopen with updated code base against Master branch. Thanks.